### PR TITLE
refactor(esl-share): rename esl-share-list with esl-share

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

**Review by ala-n — APPROVED** [review]
Also it looks like an update for file `src/modules/esl-share/core/esl-share-list.ts` is missed it is not mentioned in the change list

File: src/modules/esl-share/core/esl-share.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+esl-share[mode="list"] {
```
[the remark this answers is not in the document]
Comment (dshovchko) [reply src/modules/esl-share/core/esl-share.less:1]: Don't worry! There will be an empty element when JS is not loaded.

**Review by NastaLeo — APPROVED** [review]

File: pages/views/examples/share.njk [hunk 2]
```diff
@@ -23,47 +23,47 @@ aside:
...
   <h3>Displays list of share buttons with empty list</h3>
   <p>Shows empty component without buttons.</p>
-  <esl-share-list list=""></esl-share-list>
+  <esl-share list=""></esl-share>
   <hr>
 
   <h3>Displays list of share buttons via incorrect list</h3>
   <p>Shows buttons only with the correct names.</p>
-  <esl-share-list list="twittor  copi"></esl-share-list>
+  <esl-share list="twittor  copi"></esl-share>
```
Comment (Natalie-Smirnova) [inline pages/views/examples/share.njk:61]: And here?

**Review by Natalie-Smirnova — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Review by HenadzV — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

File: src/modules/esl-share/core/esl-share.shape.ts [hunk 3]
```diff
@@ -1,27 +1,29 @@
 import type {ESLBaseElementShape} from '../../esl-base-element/core/esl-base-element.shape';
-import type {ESLShareList} from './esl-share-list';
+import type {ESLShare} from './esl-share';
 
 /**
  * Tag declaration interface of ESL Share List element
```
Comment (yadamskaya) [inline src/modules/esl-share/core/esl-share.shape.ts:5]:
```suggestion
 * Tag declaration interface of ESL Share element
```

File: pages/views/examples/share.njk [hunk 4]
```diff
@@ -23,47 +23,47 @@ aside:
...
   <h3>Displays list of share buttons of group 'demo'</h3>
   <p>Shows buttons from a specified group.</p>
-  <esl-share-list list="group:demo"></esl-share-list>
+  <esl-share list="group:demo"></esl-share>
   <hr>
 
   <h3>Displays list of share buttons of group 'alternative'</h3>
   <p>Shows buttons from a specified group.</p>
-  <esl-share-list list=" group:alternative  "></esl-share-list>
+  <esl-share list=" group:alternative  "></esl-share>
```
[the remark this answers is not in the document]
Comment (dshovchko) [reply pages/views/examples/share.njk:31]: Yes, checking for worse cases

File: pages/views/examples/share.njk (hunk 2 above)
Comment (dshovchko) [reply pages/views/examples/share.njk:61]: Yes, checking for worse cases

**Pull request merged**